### DISCUSSION TITLE
It is possible to change the font of CTkTabview?

### DISCUSSION THREAD

**Discussion by OSD1015** [TomSchimansky/CustomTkinter#1974] [Q&A]
It is possible to change the font of CTkTabview? Because the font shown on the tabs is not a regular font

**Comment by zSynctic** [comment]
You can do that but you will have to change the font for each button of the ctktabview, here is the solution:
```
self.tabview.add("Home")
self.tabview._segmented_button._buttons_dict["you need to put the name of the button you added here in my case its Home"].configure(font=("Roboto Medium", -14))
```
